doc/user_guide/Custom_Interactivity.rst

```diff
@@ -2,4 +2,5 @@ Custom Interactivity
 ____________________
 
 .. notebook:: holoviews ../../examples/user_guide/12-Custom_Interactivity.ipynb
+    :skip_execute: True
```

Comment:
Are you sure you don't want *any* output (e.g print output)? You can use the skip_output option if you add a comment line 'When run live, this cell's output should match the behavior of the GIF below' into the code cells that should be skipped.